core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -382,6 +390,30 @@ object DumpLogSegments {
     }
   }
 
+  val metadataRecordSerde = new MetadataRecordSerde()
```

Comment:
nit: may as well move this into `ClusterMetadataLogMessageParser` since we don't need it anywhere else